packages/foam-cli/test/write-file-to-disk.test.ts

```diff
@@ -0,0 +1,19 @@
+
+import { writeFileToDisk } from '../src/utils/write-file-to-disk'
+import * as fs from 'fs';
+import * as path from 'path';
+
+describe('writeFileToDisk', () => {
+  it('should overrwrite existing file in the disk with the new data', async () => {
+    const existingData = `content in the existing file`;
+    const newData = `content in the new file`;
+    const fileUri = path.join(__dirname, 'test-file.md');
```

Comment:
If we don't mock the file system operations, at minimum we should write to a temp directory, so we don't pollute the active git workspace in case cleanup fails.